tests/integration/test_observability.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023 Canonical, Ltd.
+#
+
+import config
+import pytest
+from pytest_operator.plugin import OpsTest
+
+
+@pytest.mark.abort_on_fail
+async def test_observability_metrics(e: OpsTest):
+    await e.model.deploy(
+        config.MK8S_CHARM,
+        application_name="microk8s",
+        channel=config.MK8S_CHARM_CHANNEL,
+        constraints=config.MK8S_CONSTRAINTS,
+    )
+    await e.model.wait_for_idle(["microk8s"])
+
+    await e.model.deploy(
+        config.MK8S_GRAFANA_AGENT_CHARM,
+        channel=config.MK8S_GRAFANA_AGENT_CHANNEL,
+        application_name="grafana-agent",
+    )
+    await e.model.relate("microk8s", "grafana-agent")
+
+    # TODO(neoaggelos): enable tests after required issues are fixed
+    # - https://github.com/simskij/grafana-agent/issues/22
```

Comment:
Why is this issue against a user's repo? I see it closed, but is it addressed on the official grafana-agent repo?